<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="UDS-USER-69" failures="1" tests="1" name="Default test(failed)" time="11.476" errors="0" timestamp="2022-05-15T12:39:57 EEST">
  <testcase classname="TestMobexo.Reruntests" name="Test3" time="4.324">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestMobexo.Reruntests.Test3(Reruntests.java:71)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="TestMobexo.Reruntests" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase classname="TestMobexo.Reruntests" name="Test2" time="0.0">
    <ignored/>
  </testcase> <!-- Test2 -->
  <testcase classname="TestMobexo.Reruntests" name="Test4" time="0.0">
    <ignored/>
  </testcase> <!-- Test4 -->
</testsuite> <!-- Default test(failed) -->
